docs/user/kafka/commandline.rst

```diff
@@ -96,6 +96,77 @@ The ``--delay`` argument should be specified as a duration, in plain language. F
 or ``1 second``. The ingest will pause after creating each ``SimpleFeature`` for the specified delay.
 This can be used to simulate a live data stream.
 
+``playback``
+^^^^^^^^^^
```

Comment:
```suggestion
``playback``
^^^^^^^^^^^^
```
can't quite tell if this lines up with the formatting... i think the suggestion is correct